.gitignore

```diff
@@ -39,3 +39,11 @@ firebase-sw-private.js
 cypress.json
 cypress/screenshots
 cypress/videos
+
+.vs/VSWorkspaceState.json
+.vs/codelabz
+.vs/
+.vs/.vs/slnx.sqlite
+.vs/Codelabz/config/applicationhost.config
+.vs/Codelabz/v16/.suo
```

Comment:
Wouldn' just adding .vs/ ignore the whole directory🤔?

Reply:
@shehand Ideally it should be, but I could not get this behavior why it was not ignoring my files under the directory. For my first PR in CodeLabz, I tried to do it by just adding ./vs but in the end, I had to clean the cache and add specific files in gitignore to ignore those files.

Reply:
Have you tried `git rm -r --cached .vs` ?

Reply:
@shehand Yes, I tried this for the earlier PR

Reply:
@shehand what should I do for this?